src/ModelContextProtocol.Core/Server/AIFunctionMcpServerPrompt.cs

```diff
@@ -67,7 +67,7 @@ private static AIFunctionFactoryOptions CreateAIFunctionFactoryOptions(
         MethodInfo method, McpServerPromptCreateOptions? options) =>
         new()
         {
-            Name = options?.Name ?? method.GetCustomAttribute<McpServerPromptAttribute>()?.Name,
+            Name = options?.Name ?? method.GetCustomAttribute<McpServerPromptAttribute>()?.Name ?? AIFunctionMcpServerTool.DeriveName(method, JsonNamingPolicy.CamelCase),
```

Comment:
Not that I necessary see a benefit in doing this, but is making the naming policy configurable something we would want to do here?

Reply:
I don't think so. I think we have a default and you can override with whatever name you want. That's what we already have, this is just changing what is that default.